styles/duolingo/catppuccin.user.css

```diff
@@ -0,0 +1,359 @@
+    --color-unknown-0047a4: var(--surface1);
+    --color-unknown-0087d0: var(--overlay0);
+    --color-unknown-00aff9: var(--sapphire);
+    --color-unknown-013047: var(--base);
+    --color-unknown-048fd1: var(--sapphire);
+    --color-unknown-0e0f10: var(--crust);
+    --color-unknown-0e3d79: var(--surface1);
+    --color-unknown-172071: var(--surface0);
+    --color-unknown-280378: var(--surface0);
+    --color-unknown-3ebbf6: var(--sapphire);
+    --color-unknown-655ebb: var(--overlay1);
+    --color-unknown-696cee: var(--blue);
+    --color-unknown-7c0000: var(--base);
+    --color-unknown-89e219: var(--green);
+    --color-unknown-935051: var(--surface2);
+    --color-unknown-959595: var(--overlay1);
+    --color-unknown-a2a2a2: var(--overlay2);
+    --color-unknown-a3dbeb: var(--sky);
+    --color-unknown-a4dffb: var(--sky);
+    --color-unknown-aaa: var(--overlay2);
+    --color-unknown-d087ff: var(--mauve);
+    --color-unknown-d9d9d9: var(--rosewater);
+    --color-unknown-ddd: var(--rosewater);
+    --color-unknown-de8029: var(--peach);
+    --color-unknown-e3e3e3: var(--rosewater);
+    --color-unknown-e4ffff: var(--text);
+    --color-unknown-ed8c01: var(--peach);
+    --color-unknown-f3484e: var(--red);
+    --color-unknown-f4fafe: var(--rosewater);
+    --color-unknown-fbdec5: var(--flamingo);
+    --color-unknown-ffc700: var(--peach);
+    --color-unknown-fff2aa: var(--yellow);
+    --color-unknown-fffbef: var(--rosewater);
+  }
+
+  #catppuccin(@lookup, @accent) {
+    --rosewater: @catppuccin[@@lookup][@rosewater];
+    --flamingo: @catppuccin[@@lookup][@flamingo];
+    --pink: @catppuccin[@@lookup][@pink];
+    --mauve: @catppuccin[@@lookup][@mauve];
+    --red: @catppuccin[@@lookup][@red];
+    --maroon: @catppuccin[@@lookup][@maroon];
+    --peach: @catppuccin[@@lookup][@peach];
+    --yellow: @catppuccin[@@lookup][@yellow];
+    --green: @catppuccin[@@lookup][@green];
+    --teal: @catppuccin[@@lookup][@teal];
+    --sky: @catppuccin[@@lookup][@sky];
+    --sapphire: @catppuccin[@@lookup][@sapphire];
+    --blue: @catppuccin[@@lookup][@blue];
+    --lavender: @catppuccin[@@lookup][@lavender];
+    --text: @catppuccin[@@lookup][@text];
+    --subtext1: @catppuccin[@@lookup][@subtext1];
+    --subtext0: @catppuccin[@@lookup][@subtext0];
+    --overlay2: @catppuccin[@@lookup][@overlay2];
+    --overlay1: @catppuccin[@@lookup][@overlay1];
+    --overlay0: @catppuccin[@@lookup][@overlay0];
+    --surface2: @catppuccin[@@lookup][@surface2];
+    --surface1: @catppuccin[@@lookup][@surface1];
+    --surface0: @catppuccin[@@lookup][@surface0];
+    --base: @catppuccin[@@lookup][@base];
+    --mantle: @catppuccin[@@lookup][@mantle];
+    --crust: @catppuccin[@@lookup][@crust];
+    --accent-color: @catppuccin[@@lookup][@@accent];
+
+    color-scheme: if(@lookup = latte, light, dark);
+
+    ::selection {
+      // https://github.com/less/less.js/issues/3611
+      // background-color: fade(@accent-color, 30%);
+    }
```

Comment:
?

Reply:
Solved the issue using the [#rgbify function](https://github.com/catppuccin/userstyles/blob/main/docs/tips-and-tricks.md#how-do-i-set-a-variable-to-rgb-values) that you sugested on Discord